assets/js/components/ClusterDetails/AscsErsClusterDetails.jsx

```diff
@@ -2,27 +2,78 @@ import React, { useState, useEffect } from 'react';
 
 import PageHeader from '@components/PageHeader';
 import BackButton from '@components/BackButton';
-
+import Table from '@components/Table';
 import ListView from '@components/ListView';
 import ProviderLabel from '@components/ProviderLabel';
 import DottedPagination from '@components/DottedPagination';
+import HostLink from '@components/HostLink';
 
 import ChecksComingSoon from '@static/checks-coming-soon.svg';
 
 import SBDDetails from './SBDDetails';
+import SiteDetails from './SiteDetails';
 import StoppedResources from './StoppedResources';
 
+const nodeDetailsConfig = {
+  usePadding: false,
+  columns: [
+    {
+      title: 'Hostname',
+      key: '',
+      render: (_, { id, name }) => <HostLink hostId={id}>{name}</HostLink>,
+    },
+    {
+      title: 'Role',
+      key: 'roles',
+      render: (content) =>
+        content?.map((role) => role.toUpperCase()).join(', '),
+    },
+    {
+      title: 'Virtual IP',
+      key: 'virtual_ips',
+      className: 'table-col-m',
+      render: (content) => content?.join(', '),
+    },
+    {
+      title: 'Filesystem',
+      key: 'filesystems',
+      className: 'table-col-m',
+      render: (content) => content?.join(', '),
+    },
+    {
+      title: '',
+      key: '',
+      className: 'table-col-xs',
+      render: (_, item) => {
+        const { attributes, resources } = item;
+        return <SiteDetails attributes={attributes} resources={resources} />;
+      },
+    },
+  ],
+};
+
 function AscsErsClusterDetails({
   clusterName,
   cibLastWritten,
   provider,
+  hosts,
   details,
 }) {
+  const [sapSystems, setSapSystems] = useState([]);
   const [currentSapSystem, setSapSystem] = useState(null);
 
   useEffect(() => {
-    setSapSystem(details?.sap_systems[0]);
-  }, [details]);
+    const enrichedSapSystems = details?.sap_systems.map((sapSystem) => ({
+      ...sapSystem,
+      nodes: sapSystem?.nodes.map((node) => ({
+        ...node,
+        ...hosts.find(({ hostname }) => hostname === node.name),
+      })),
+    }));
+
+    setSapSystems(enrichedSapSystems);
+    setSapSystem(enrichedSapSystems[0]);
```

Comment:
Do we always have at least one element in `enrichedSapSystems`? 
Or it can be an empty list? If so, we should consider the case.

I didn't try this myself, though.

Reply:
Yes, we should always have one sap system. This is a hard requirement in the backend. 
Otherwise, the cluster is not identified as `ASCS/ERS` cluster. It would be an unknown cluster, which shouldn't have this page, and we should go to the `Unknown cluster type` view.

Anyway, in a pure way, if the list is empty, we simply have the same view with many empty fields. Nothing would break